opendocument-app/OpenDocument.core
# refactor travis ci

- fetch from OpenDocument.test
- run unit tests
- handle encrypted files
- use html tidy
- compare latest build to reference build (like master)
- see #75 for reference
